src/onboarding/ha-onboarding.js

```diff
@@ -42,6 +42,7 @@ class HaOnboarding extends localizeLiteMixin(PolymerElement) {
         value='{{_name}}'
         required
         auto-validate
+        autocapitalize='words'
```

Comment:
I think we should use `on`/`sentences` here instead. There is an ongoing issue since iOS 8 with `words` that causes the first 2 letters to be capitalized instead of every word (I've confirmed this is still an issue in 11.4). That is much more annoying IMO than having to manually capitalize extra words

Reply:
Really..? Since the names I use are one word anyway I'm OK with this